[7983] Change apostrophes from straight to curly style (#5043)
DFE-Digital/register-trainee-teachers: docs/machine-setup.md

When the versions are updated in main run `asdf install` again to update your
installation.

> (We don’t mandate asdf, you can use other tools if you prefer.)

Reviewer: ```suggestion
(We do not mandate asdf, you can use other tools if you prefer.)
```
